src/pools/abstract-pool.ts

```diff
@@ -100,9 +100,9 @@ class PoolEmitter extends EventEmitter {}
  */
 export abstract class AbstractPool<
   Worker extends IWorker,
-  Data = unknown,
-  Response = unknown
-> implements IPool<Data, Response> {
+  Data extends JSONValue = JSONValue,
```

Comment:
@pioardi told me that we specially don't want that on `AbstractPool`

Reply:
Yes I would prefer to have it much more open